src/Database/PostgreSQL/PQTypes/Checks.hs

```diff
@@ -381,6 +383,9 @@ checkDBStructure tables = fmap mconcat . forM tables $ \table ->
         checkPrimaryKey mdef mpk = mconcat [
             checkEquality "PRIMARY KEY" def (map fst pk)
           , checkNames (const (pkName tblName)) pk
+          , if (EnforceMandatoryPrimaryKeys `elem` options)
```

Comment:
I'm wondering whether we should ditch this pattern. More standard way is to do something like

```
data CheckOptions = CheckOptions {
    chkOptEnforceMandatoryPrimaryKeys :: Bool,
    ...
}
```

Reply:
I see, you already wrote above that you want to do this.